### ISSUE TITLE
Test for WCAG success criterion 3.2.3: Consistent navigation

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#188]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-locations.html
Navigational mechanisms that are repeated on multiple Web pages within a set of Web pages occur in the same relative order each time they are repeated, unless a change is initiated by the user.
It is important to note that the use of the phrase "same order" in this section is not meant to imply that subnavigation menus cannot be used or that blocks of secondary navigation or page structure cannot be used. Instead, this Success Criterion is intended to assist users who interact with repeated content across Web pages to be able to predict the location of the content they are looking for and find it more quickly when they encounter it again. 
## Applicable

Set of web pages, 
## Not applicable

Only one page.
## Scope selector

Navigation mechanisms (search form, skip links, menu's)
## Examples
- A consistently located control: A search field is the last item on every Web page in a site. users can quickly locate the search function.
- An expanding navigation menu: A navigation menu includes a list of seven items with links to the main sections of a site. When a user selects one of these items, a list of subnavigation items is inserted into the top-level navigation menu.
- Consistently positioned skip navigation controls: A "skip navigation" (or "skip to main content") link is included as the first link on every page in a Web site. The link allows users to quickly bypass heading information and navigational content and begin interacting with the main content of a page.
- Skip to navigation link: Navigational content is consistently located at the end of each page in a set of Web pages. A "skip to navigation" link is consistently located at the beginning of each page so that keyboard users can easily locate it when needed. 
## Failures
- [ ] F66: Presenting navigation links in a different relative order on different pages 
## Techniques
- [ ] G61: Presenting repeated components in the same relative order each time they appear ( #189 )
## Relationship between techniques and failures
- F66 is acounterpart for G61 for menu's (test 3)
- as G61 is the only technology, if G61 fails, this success criterion fails and vv
